CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ Changelog
   :class:`~cryptography.hazmat.primitives.hashes.SHA3_384`, and
   :class:`~cryptography.hazmat.primitives.hashes.SHA3_512` when using OpenSSL
   1.1.1.
+* Added initial support for parsing PKCS12 files with
+  :func:`~cryptography.hazmat.primitives.serialization.pkcs12.load_load_key_and_certificates_from_pkcs12`.
```

Comment:
I feel like `pkcs12` only needs to be in the name once, pick one :-)

Reply:
You couldn't have expressed this opinion in IRC when you said you didn't care before? 😝

Reply:
I probably could have, truth be told. Sorry 'bout that!